charts/opentelemetry-operator/values.yaml

```diff
@@ -204,11 +204,14 @@ manager:
 
   ## Container specific securityContext
   ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#set-the-security-context-for-a-container
-  securityContext: {}
-  # allowPrivilegeEscalation: false
-  # capabilities:
-  #   drop:
-  #   - ALL
+  securityContext:
```

Comment:
I believe this would be considered a breaking change and we dont want to do that. I agree that we can make the template/test files use a security context, but we should not set a default value here.

Reply:
Can you be a little more specific on what you believe setting the `securityContext` breaks, because it is not immediately obvious to me. Existing installation should continue work as the operator does not need does extra privileges or capabilities. I have intentionally not made the change for the collector as some of the collector components does need elevated privileges.

Reply:
> Existing installation should continue work as the operator does not need does extra privileges or capabilities.

@jaronoff97 can you verify this is true. @jenrik have you confirmed the operator installs and operates correctly with this security context?

When I see changes to default values in our values.yaml I normally consider them breaking changes until we can prove that they aren't. We are very careful with how we change default values as it can cause unexpected problems for users performing a `helm upgrade`.

Reply:
This should probably work fine or get caught by our tests in the CI. @jenrik can you pls [generate examples](https://github.com/open-telemetry/opentelemetry-helm-charts/blob/main/charts/opentelemetry-operator/CONTRIBUTING.md) so we can run tests and have CI pass?

Reply:
@TylerHelmuth I have deployed my modified chart to my personal Kubernetes cluster and found no breakage.

@Allex1 generation of examples done